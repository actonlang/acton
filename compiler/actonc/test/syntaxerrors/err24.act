# Double colon in type annotation
def typed_func() :: int:
    return 42