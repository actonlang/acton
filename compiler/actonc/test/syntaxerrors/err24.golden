Building file test/syntaxerrors/err24.act using temporary scratch directory
[error Parse error]: unexpected ": int:<newline>    ret"
                     expecting end of line or simple statement

     +--> test/syntaxerrors/err24.act@2:19-2:20
     |
   2 | def typed_func() :: int:
     :                   ^ 
     :                   `- unexpected ": int:<newline>    ret"
     :                      expecting end of line or simple statement
     :                      
-----+
